tests/systemtests/rest_cli.go

```
@@ -66,14 +66,23 @@ func GetRequest(t *testing.T, url string) []byte {
 
 func GetRequestWithHeaders(t *testing.T, url string, headers map[string]string, expCode int) []byte {
 	t.Helper()
+	
```

golangci-lint (1.61.0)

File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) --custom-order

(gci)